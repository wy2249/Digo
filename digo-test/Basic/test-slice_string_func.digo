func foo(s []string) []string {
    s2 := append(s, "haha")
    return s2
}

func digo_main() void {
  a := []string {"hello", "word"}
  b := foo(a)
  println("%d",len(a))
  println("%d",len(b))
  println("%s", a[1])
  println("%s", b[2])
}
