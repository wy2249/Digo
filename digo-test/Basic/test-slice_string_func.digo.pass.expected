2
3
word
haha
